README.md

```diff
@@ -40,6 +40,7 @@ More information may be found in its [README file](./staging/README.md).
 Here is the list of device backends in **staging**:
 
 - [Sound](https://github.com/rust-vmm/vhost-device/blob/main/staging/vhost-device-sound/README.md)
+- [VIDEO](https://github.com/rust-vmm/vhost-device/blob/main/staging/vhost-device-video/README.md)
```

Comment:
```suggestion
- [Video](https://github.com/rust-vmm/vhost-device/blob/main/staging/vhost-device-video/README.md)
```

Reply:
I wonder if it wouldn't make more sense to make `Sound` all uppercase. All non-staging crates links are all uppercase, and the example commented below also hints for uppercase.

That said, I do not have a problem changing `Video` to uppercase.

Reply:
> I wonder if it wouldn't make more sense to make `Sound` all uppercase. All non-staging crates links are all uppercase, and the example commented below also hints for uppercase.

Yes. @epilys Can you please fix that ?

Reply:
The others are all acronyms, which is why I think they are all capitalized.
VSOCK is an exception, but I think this comes from AF_VSOCK.

Anyway, I don't have a strong opinion on this (in fact, I've already accepted this PR ;-).

Reply:
> The others are all acronyms, which is why I think they are all capitalized.

I surely missed that point and it does make sense. I think yes, Sound and Video should be write like this only :)

Reply:
Understood :)

Then I'll update Video as suggested!

Reply:
Quoting the spec[^1]:

Matching with our naming:

>  5.18 GPIO Device
>  5.16 I2C Adapter Device
>  5.17 SCMI Device
>  5.6 SCSI Host Device
>  5.14 Sound Device


Not matching:

RNG:

>  5.4 Entropy Device

VSOCK:

>  5.10 Socket Device

Feel free to open a separate issue if you believe we should change those two.


[^1]: https://docs.oasis-open.org/virtio/virtio/v1.2/csd01/virtio-v1.2-csd01.html#x1-3650007